src/index.ts

```diff
@@ -110,7 +111,7 @@ try {
                 </ul>`;
 
     const indexMarkup = generateHTML(menu, path.basename(input), true, lang, indexCSS);
-    fs.writeFileSync(`${output}/index.html`, indexMarkup, { flag: 'w' });
+    fs.writeFile(`${output}/index.html`, indexMarkup, { flag: 'w' });
```

Comment:
This needs a callback if you don't have it be async (e.g., to deal with the error case).